tests/Sort.test.ts

```diff
@@ -222,45 +287,179 @@ describe('Sort', () => {
         expect(
             Sort.by(
                 {
-                    sorting: [{ property: 'description', reverse: false }],
+                    sorting: [
+                        {
+                            property: 'description',
+                            reverse: false,
+                            propertyInstance: 0,
+                        },
+                    ],
                 },
                 [two, one, five, four, three],
             ),
         ).toEqual(expectedOrder);
     });
+});
 
-    it('sorts correctly by tag, done', () => {
-        const one = fromLine({
-            line: '- [ ] a #someday #home 📅 1970-01-01 ✅ 1971-01-01',
-            path: '',
-        });
-        const two = fromLine({
-            line: '- [ ] #task b #someday #home 📅 1970-01-02 ✅ 1971-01-02',
-            path: '',
-        });
-        const three = fromLine({
-            line: '- [ ] c #next #home 📅 1970-01-02 ✅ 1971-01-01',
-            path: '',
-        });
-        const four = fromLine({
-            line: '- [ ] d #urgent #home 📅 1970-01-02 ✅ 1971-01-03',
-            path: '',
-        });
-        const five = fromLine({
-            line: '- [ ] e 📅 1970-01-02 ✅ 1971-01-03',
-            path: '',
-        });
-        const expectedOrder = [three, one, two, four, five];
+describe('Sort by tags', () => {
+    it('should sort correctly by tag defaulting to first with no global filter', () => {
+        const t1 = fromLine({ line: '- [ ] a #aaa #jjj' });
+        const t2 = fromLine({ line: '- [ ] a #ggg #ccc' });
+        const t3 = fromLine({ line: '- [ ] a #bbb #iii' });
+        const t4 = fromLine({ line: '- [ ] a #fff #aaa' });
+        const t5 = fromLine({ line: '- [ ] a #ccc #bbb' });
+        const t6 = fromLine({ line: '- [ ] a #eee #fff' });
+        const t7 = fromLine({ line: '- [ ] a #ddd #ggg' });
+        const t8 = fromLine({ line: '- [ ] a #hhh #eee' });
+        const t9 = fromLine({ line: '- [ ] a #iii #ddd' });
+        const t10 = fromLine({ line: '- [ ] a #jjj #hhh' });
+        const expectedOrder = [t1, t3, t5, t7, t6, t4, t2, t8, t9, t10];
+        expect(
+            Sort.by(
+                {
+                    sorting: [
+                        {
+                            property: 'tag',
+                            reverse: false,
+                            propertyInstance: 0,
+                        },
+                    ],
+                },
+                [t1, t2, t3, t4, t5, t6, t7, t8, t9, t10],
+            ),
+        ).toEqual(expectedOrder);
+    });
+    it('should sort correctly by second tag with no global filter', () => {
```

Comment:
Test coverage could be improved be making this one do a reverse sort order

Reply:
Makes sense :+1:

Reply:
Added reversed tests